### ISSUE TITLE
Hiding some stats fields me the graph

### ISSUE THREAD

**Issue by Jacopx** [vsimakhin/web-logbook#114]
Hi, thanks for your work.

This is a (partially, look only at the first 3 colums) correct visualization of my stats:

![Screenshot 2023-03-27 at 11-05-19 Stats](https://user-images.githubusercontent.com/14002538/227895676-08ad8fe4-4179-4ea8-868e-5461bb16e8a9.png)

hiding the ME fields from the settings cause a shift and an error to the visualization:

![Screenshot 2023-03-27 at 11-05-36 Stats](https://user-images.githubusercontent.com/14002538/227896052-6494e553-1375-47a9-a18b-53923a006b83.png)

The ME value should be zero, this doesn't work for any fields.

**Comment by vsimakhin** [comment]
right... I need to review how I build the chart. Thanks for the report!
